src/scripts/quests/QuestLineHelper.ts

```diff
@@ -2189,7 +2189,55 @@ class QuestLineHelper {
 
         App.game.quests.questLines().push(princessDiancieQuestLine);
     }
+    public static createClashOfAgesQuestLine() {
+        const clashOfAgesQuestLine = new QuestLine('Clash of Ages', 'Hoopa is up to something mischievous...', new GymBadgeRequirement(BadgeEnums.Elite_KalosChampion) , GameConstants.BulletinBoards.Kalos);
 
+        const catchHoopa = new CaptureSpecificPokemonQuest('Hoopa', 'Catch Hoopa to learn more.', 1, false, 0, undefined);
+        clashOfAgesQuestLine.addQuest(catchHoopa);
+
+        const talkToBaraz1 = new TalkToNPCQuest(Baraz1, 'Talk to Baraz in Kiloude City.');
+        clashOfAgesQuestLine.addQuest(talkToBaraz1);
+
+        const hoopaBeatPsychic = new CustomQuest(100, 0, 'Defeat 100 Psychic-type Pokémon.', () => {
+            return pokemonMap.filter(p => p.type.includes(PokemonType.Psychic)).map(p => App.game.statistics.pokemonDefeated[p.id]()).reduce((a,b) => a + b, 0);
+        });
+        clashOfAgesQuestLine.addQuest(hoopaBeatPsychic);
+
+        const talkToBaraz2 = new TalkToNPCQuest(Baraz2, 'That didn\'t work. Talk to Baraz in Kiloude City.');
+        clashOfAgesQuestLine.addQuest(talkToBaraz2);
+
+        const hoopaCatchPsychic = new CapturePokemonTypesQuest(100, undefined, PokemonType.Psychic);
+        clashOfAgesQuestLine.addQuest(hoopaCatchPsychic);
+
+        const talkToBaraz3 = new TalkToNPCQuest(Baraz3, 'That didn\'t work either. Talk to Baraz in Kiloude City.');
+        clashOfAgesQuestLine.addQuest(talkToBaraz3);
+
+        const catch100Hoopa = new CaptureSpecificPokemonQuest('Hoopa', 'Catch 100 Hoopa.', 1, false, 0, undefined);
```

Comment:
As I suggested it on Discord, here is the change. I made sure it worked.
```suggestion
        const hoopaID = PokemonHelper.getPokemonByName('Hoopa').id;
        const catch100Hoopa = new CustomQuest(100, 0, 'Catch 100 Hoopa.', () => 100 * (App.game.statistics.pokemonCaptured[hoopaID]() - App.game.statistics.pokemonHatched[hoopaID]()));
```